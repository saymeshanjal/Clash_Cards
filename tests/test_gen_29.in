99
4 207 5
14 162 2
14 356 2
16 259 4
4 295 5
4 234 4
16 116 3
4 111 1
10 371 1
10 356 1
8 198 5
8 372 2
12 234 5
16 313 5
4 202 5
10 284 1
14 75 2
4 348 3
6 393 1
14 293 3
12 217 3
8 396 5
10 82 1
2 1 1
2 58 3
16 246 2
2 268 2
12 389 3
4 56 1
4 270 1
12 179 5
16 294 2
12 26 1
12 35 2
14 227 5
4 99 1
4 187 2
8 257 1
6 370 2
6 99 4
2 152 3
14 327 5
4 148 1
14 153 3
10 351 1
10 1 5
8 282 4
14 69 1
14 16 4
16 114 3
12 258 3
16 169 5
16 290 1
4 305 1
16 117 3
2 186 1
10 5 3
8 147 2
10 201 5
16 272 2
12 182 2
10 61 1
8 156 2
14 141 4
6 4 3
14 372 4
8 96 1
6 116 5
16 336 2
8 279 4
6 396 1
10 41 4
14 292 5
8 50 4
4 268 3
12 51 1
6 198 4
12 193 5
6 104 3
4 251 2
4 302 1
14 162 4
4 304 3
16 220 3
8 213 3
10 354 4
4 381 5
14 298 3
2 204 2
4 302 5
6 63 2
16 373 1
12 142 3
16 278 3
8 121 1
8 29 1
16 374 2
4 53 2
10 333 1
