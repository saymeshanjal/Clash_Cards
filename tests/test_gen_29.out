0 6 0
1078 1079 1275
1078 1079 1275
3809 5406 3655
0 6 0
0 5 1
6307 6563 0
3 3 0
126 126 0
126 126 0
8 62 0
20 21 29
219 705 0
4248 8622 0
0 6 0
126 126 0
1078 1079 1275
1 5 0
10 10 0
1652 1780 0
430 494 0
8 62 0
126 126 0
1 1 0
0 2 0
4081 4082 4707
0 1 1
430 494 0
3 3 0
3 3 0
219 705 0
4081 4082 4707
443 481 0
126 170 628
987 2445 0
3 3 0
1 2 3
35 35 0
5 6 9
1 14 5
0 2 0
987 2445 0
3 3 0
1652 1780 0
126 126 0
0 252 0
8 42 20
1716 1716 0
18 2973 441
6307 6563 0
430 494 0
4248 8622 0
6435 6435 0
3 3 0
6307 6563 0
1 1 0
6 246 0
20 21 29
45 207 0
4081 4082 4707
286 287 351
126 126 0
20 21 29
910 1520 1002
0 20 0
910 1520 1002
35 35 0
1 19 0
4081 4082 4707
8 42 20
10 10 0
44 133 75
987 2445 0
8 42 20
1 5 0
462 462 0
1 14 5
219 705 0
6 14 0
1 2 3
3 3 0
910 1520 1002
1 5 0
6307 6563 0
27 43 0
44 133 75
0 6 0
1652 1780 0
0 1 1
0 6 0
5 6 9
6435 6435 0
430 494 0
6307 6563 0
35 35 0
35 35 0
4081 4082 4707
1 2 3
126 126 0
